modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -36,70 +46,131 @@ export default class ortcRTCPeerConnection extends yaeti.EventTarget {
+        // - value: Object
+        //   - rtpSender: Associated RTCRtpSender instance
+        //   - stream: Associated MediaStream instance
+        //   - ssrc: Provisional or definitive SSRC
+        //   - rtxSsrc: Provisional or definitive SSRC for RTX
+        //   - sending: Boolean indicating whether rtpSender.send() was called.
+        this._localTrackInfos = new Map();
+
+        // Ordered Map with MID as key and kind as value.
+        // @type {map<String, String>}
+        this._mids = new Map();
 
         // Remote RTCSessionDescription.
         // @type {RTCSessionDescription}
         this._remoteDescription = null;
 
-        // Set of remote MediaStreams.
-        // @type {Set<MediaStream>}
-        this._remoteStreams = new Set();
+        // Map of remote streams.
+        // - index: MediaStream.jitsiRemoteId (as signaled in remote SDP)
+        // - value: MediaStream (locally generated so id does not match)
+        // @type {map<Number, MediaStream>}
+        this._remoteStreams = new Map();
+
+        // Map with info about receiving media.
+        // - index: Media SSRC
+        // - value: Object
+        //   - kind: 'audio' / 'video'
+        //   - ssrc: Media SSRC
+        //   - rtxSsrc: RTX SSRC (may be unset)
+        //   - streamId: MediaStream.jitsiRemoteId
+        //   - trackId: MediaStreamTrack.jitsiRemoteId
+        //   - cname: CNAME
+        //   - stream: MediaStream
+        //   - rtpReceiver: Associated RTCRtpReceiver instance
+        // @type {map<Number, Object>}
+        this._remoteTrackInfos = new Map();
+
+        // Local SDP global fields.
+        this._sdpGlobalFields = {
+            id: RandomUtil.randomInt(1000000, 9999999),
+            version: 0
+        };
 
         // RTCPeerConnection signalingState.
         // @type {RTCSignalingState}
         this._signalingState = RTCSignalingState.stable;
+
+        // Create the RTCIceGatherer.
+        this._setIceGatherer(pcConfig);
+
+        // Create the RTCIceTransport.
+        this._setIceTransport(this._iceGatherer);
+
+        // Create the RTCDtlsTransport.
+        this._setDtlsTransport(this._iceTransport);
     }
 
     /**
-     * Gets the current signaling state.
+     * Current signaling state.
      * @return {RTCSignalingState}
      */
     get signalingState() {
         return this._signalingState;
     }
 
     /**
-     * Gets the current ICE gathering state.
+     * Current ICE gathering state.
      * @return {RTCIceGatheringState}
      */
     get iceGatheringState() {
         return this._iceGatheringState;
     }
 
     /**
-     * Gets the current ICE connection state.
+     * Current ICE connection state.
      * @return {RTCIceConnectionState}
      */
     get iceConnectionState() {
         return this._iceTransport.state;
     }
 
     /**
+     * Current ICE+DTLS connection state.
+     * @return {RTCPeerConnectionState}
+     */
+    get connectionState() {
```

Comment:
For the Nth time, alphabetical sort. By getter name. Except the constructor, which always goes first. It's not that hard.